composer.json

```diff
@@ -18,7 +18,7 @@
     "ext-bcmath": "*",
     "ext-simplexml": "*",
     "doctrine/annotations": "^1.6",
-    "eoneopay/standards": "dev-master",
+    "eoneopay/standards": "^1.0",
```

Comment:
Namespace should be changed to eonx-com/standards

Reply:
@ketanp77 this may not be possible, if composer starts wigging out leave it as eoneopay for now